backend/entities/views.py

```diff
@@ -70,7 +75,7 @@ def destroy(self, request: Request, *args: str, **kwargs: int) -> Response:
         instance = get_object_or_404(Organization, pk=kwargs["pk"])
         instance.delete()
         data = {"message": f'Organization {kwargs["pk"]} has been deleted successfully'}
```

Comment:
The variable data is created but it is not used. You have to add it to the response.